src/renderer/components/AccountDistribution/Row.tsx

```diff
@@ -1,105 +1,59 @@
-// @flow
-
 import React, { useCallback } from "react";
-import { getCurrencyColor } from "~/renderer/getCurrencyColor";
+// @ts-ignore
+import { getCurrencyColor } from "../../getCurrencyColor";
+// @ts-ignore
 import { getAccountName } from "@ledgerhq/live-common/lib/account";
-import type { Account, TokenAccount } from "@ledgerhq/live-common/lib/types/account";
-import type { CryptoCurrency, TokenCurrency } from "@ledgerhq/live-common/lib/types/currencies";
+// import type { Account, TokenAccount } from "@ledgerhq/live-common/lib/types/account";
+// import type { CryptoCurrency, TokenCurrency } from "@ledgerhq/live-common/lib/types/currencies";
 import { BigNumber } from "bignumber.js";
 import { useSelector } from "react-redux";
-import styled from "styled-components";
-import CounterValue from "~/renderer/components/CounterValue";
-import FormattedVal from "~/renderer/components/FormattedVal";
-import Text from "~/renderer/components/Text";
-import Ellipsis from "~/renderer/components/Ellipsis";
-import ParentCryptoCurrencyIcon from "~/renderer/components/ParentCryptoCurrencyIcon";
-import Box from "~/renderer/components/Box";
-import AccountContextMenu from "~/renderer/components/ContextMenu/AccountContextMenu";
-import { accountsSelector } from "~/renderer/reducers/accounts";
-import IconDots from "~/renderer/icons/Dots";
-import Bar from "~/renderer/components/AssetDistribution/Bar";
-import ToolTip from "~/renderer/components/Tooltip";
+import styled, { useTheme } from "styled-components";
+// @ts-ignore
+import CounterValue from "../CounterValue";
+// @ts-ignore
+import FormattedVal from "../FormattedVal";
+// @ts-ignore
+import Text from "../Text";
+// @ts-ignore
+import Ellipsis from "../Ellipsis";
+// @ts-ignore
+import ParentCryptoCurrencyIcon from "../ParentCryptoCurrencyIcon";
+// @ts-ignore
+import Box from "../Box";
+// @ts-ignore
+import AccountContextMenu from "../ContextMenu/AccountContextMenu";
+// @ts-ignore
+import { accountsSelector } from "../../reducers/accounts";
+// @ts-ignore
+import IconDots from "../../icons/Dots";
+// @ts-ignore
+import Bar from "../AssetDistribution/Bar";
+// @ts-ignore
+import ToolTip from "../Tooltip";
 
 import { useHistory } from "react-router-dom";
-import useTheme from "~/renderer/hooks/useTheme";
-import type { ThemedComponent } from "~/renderer/styles/StyleProvider";
-
-export type AccountDistributionItem = {
-  account: Account | TokenAccount,
-  distribution: number, // % of the total (normalized in 0-1)
-  amount: BigNumber,
-  currency: CryptoCurrency | TokenCurrency,
-  countervalue: BigNumber, // countervalue of the amount that was calculated based of the rate provided
-};
 
-type Props = {
-  item: AccountDistributionItem,
-  isVisible: boolean,
-};
+export interface AccountDistributionItem {
+  // [TODO] account: Account | TokenAccount;
+  account: any;
```

Comment:
i'm not sure we should merge this (lower our type coverage) **until** we migrate live-common tho?

Reply:
We should migrate from ledgerjs -> ledger-live-common -> ledger-live-desktop/mobile I think.